tests/test_merge.py

```diff
@@ -21,8 +21,8 @@ def setUp(self) -> None:
     def test_merge_multiple_inputs(self):
         """Test merging of multiple msdfs."""
         merged_msdf = merge_msdf(*self.msdfs)
-        self.assertEqual(123, len(merged_msdf.df))
+        self.assertEqual(170, len(merged_msdf.df))
```

Comment:
very very big change. Is this expected?

Reply:
merge_msdf should not reconcile by default.. Is it?

Reply:
reconcile is True by default.

Reply:
Ok it should not be, can we make it false? during a simple merge, no information should be lost. Make sure that all the API methods are also not set to TRUE by default..

Reply:
Done, test numbers will change for `test_reconcile` and `test_merge`.